src/BullOak.Repositories.EventStore.Test.Integration/Specification/SaveEventStream.feature

```diff
@@ -66,3 +66,13 @@ Scenario: Write after a soft deleted stream should succeed
 	And I load my entity
 	Then the load process should succeed
 	And HighOrder property should be 9
+
+Scenario: Write after a soft delete by event for a stream should succeed
```

Comment:
I think need one more scenario to test what happens when we try to re-create stream with the same Id